tests/test-distribution/test-distribution-common/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -1174,4 +1174,107 @@ public void testDryRunProperties() throws Exception
             }
         }
     }
+
+    @Test
+    public void testFastCGIProxying() throws Exception
+    {
+        String jettyVersion = System.getProperty("jettyVersion");
+        JettyHomeTester distribution = JettyHomeTester.Builder.newInstance()
+            .jettyVersion(jettyVersion)
+            .mavenLocalRepository(System.getProperty("mavenRepoPath"))
+            .build();
+
+        List<String> args1 = List.of("--add-modules=resources,http,fcgi-proxy,core-deploy");
```

Comment:
This one tests `fcgi-proxy`, but what about `fcgi` module itself?

Also, what's all of the "fcgi" stuff in the `testBeforeDirectiveInModule` method in this class about? (it's also a `@Disabled` test)

Reply:
The server `fcgi` module is really only for testing.
It is available as a `*.mod` and just add the relevant jars to the class-path, but has no XML file associated, again because it's mostly for testing.

The disabled tests will be re-enabled in a different PR I'm working on.